syntax = "proto3";
package UDP;

message FileData{
	int32 index = 1;
	string data = 2;
}

message DataRcvAck{
	int32 index = 1;
}

message DataRcvDone{
	string data = 1;
}
